### ISSUE TITLE
Prying component entities works while pulling

### ISSUE THREAD

**Issue by lzk228** [space-wizards/space-station-14#30157]
1. Pull entity with PryingComponent
2. use virtual item in your hand like an actual tool
3. (optional) pry captain room with dead tarantula :trollface:

https://github.com/user-attachments/assets/b316d69f-7154-4bc8-ae3c-6336ff83d96a

**Comment by Plykiya** [comment]
this is also why you can use mobs as a weapon, right?

**Comment by ShadowCommander** [comment]
Does this still happen before #29703?

**Comment by lzk228** [comment]
> Does this still happen before #29703?

yes, it works in previous commits

**Comment by lzk228** [comment]
> this is also why you can use mobs as a weapon, right?

i dont think so, since mobs with MeleeWeapon actually reasonable can be used as weapon while held, and they cant be used like weapon while pulled, so... not relatable

**Comment by osjarw** [comment]
Apparently you can also use a cleanbot as a mop, and in my currently unsubmitted PR a medibot as a syringe.
